File: app/src/main/java/za/co/kernelpanic/cloudy/data/Weather.java
package za.co.kernelpanic.cloudy.data;


import com.google.gson.annotations.SerializedName;

public class Weather {

    private String main;
    private String description;

    @SerializedName("id")
    private int iconId;

    public String getMain() {
        return main;
    }

    public String getDescription() {
        return description;
    }

    public int getIconId() {
        return iconId;
    }


}

File: app/src/main/java/za/co/kernelpanic/cloudy/data/Weather.kt
package za.co.kernelpanic.cloudy.data

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass (generateAdapter = true)
data class Weather (val main: String? = "",
                    val description: String? = "",
                    @Json(name = "id") val iconId: Int? = 0)